src/component/Header/MenuBar/Explore.jsx

```diff
@@ -0,0 +1,54 @@
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import * as React from 'react';
```

Comment:
dont import *... import only what you're gonna use (useState)

Reply:
okey, i will change it.